totalRP3/resources/imageList.lua

```diff
@@ -17,6 +17,11 @@
 --- limitations under the License.
 ----------------------------------------------------------------------------------
 
+local version = select(4, GetBuildInfo());
+if version < 80100 then
```

Comment:
This should probably test the WOW_PROJECT_ID constant to be consistent with other checks.